mslice/plotting/plot_window/cut_plot.py

```diff
@@ -321,15 +325,28 @@ def set_is_icut(self, is_icut):
     def is_icut(self):
         return self._is_icut
 
-    def update_bragg_peaks(self):
+    def _get_overplot_datum(self):
+        return np.nanmedian([line.get_ydata() for line in self._canvas.figure.gca().get_lines()
+                             if not self._cut_plotter_presenter.is_overplot(line)])
+
+    def update_bragg_peaks(self, refresh=None):
+        refresh = False if refresh is None else True
```

Comment:
Why not just set `refresh=False` by default?

Reply:
This sounds like good advice for mutable objects and functions as it says in the article, but I would say that it has no benefit for things like booleans

Reply:
Point taken, perhaps I should have read more than the first two paragraphs of the article!